labelbox/schema/project.py

```diff
@@ -1655,6 +1655,73 @@ def __check_data_rows_have_been_processed(
         return response["queryAllDataRowsHaveBeenProcessed"][
             "allDataRowsHaveBeenProcessed"]
 
+    def get_overview(self, with_issues: bool = False) -> Dict:
+        """ Return the number of data rows per task queue, and issues of a project
+            Equivalent of the Overview tab of a project
+
+                Args:
+                    with_issues: (optional) boolean to include issues in the overview
+                Returns:
+                    Dictionary with the following structure:
+                    {
+                        'Labeled': (int) Number of labeled data rows,
+                        'InReview': (dict) Name of the in review queue and associated number of data rows
+                        'InRework': (int) Number of data rows in rework,
+                        'Done': (int) Number of completed data rows,
+                        'ToLabel': (int) Number of data rows to label,
+                        'AllDataRows': (int) Total number of data rows,
+                        'Issues': (optional) Number of issues
+                    }
+                """
+
+        query = """query ProjectGetOverviewPyApi($projectId: ID!) {
+            project(where: { id: $projectId }) {      
+            workstreamStateCounts {
+                state
+                count
+            }
+            taskQueues {
+                queueType
+                name
+                dataRowCount
+            }
+            issues {
+                totalCount
+            }
+            completedDataRowCount
+            }
+        }
+        """
+
+        # Must use experimental to access "issues"
+        result = self.client.execute(query, {"projectId": self.uid},
+                                     experimental=True)["project"]
+
+        overview = {
+            utils.snake_case(st["state"]): st["count"]
+            for st in result["workstreamStateCounts"]
```

Comment:
Is `workstreamStateCounts` always guaranteed to be there? The same question for other keys